config/locales/en.yml

```diff
@@ -104,42 +149,131 @@ en:
     verifications:
       edit:
         title: "Application submitted"
-        app_number_header: "Application Number"
-        app_number_desc: "The client should write this number down. They will need it to pick up the benefit in person."
+        verifications:
+          title: "Verification documents"
+          message: "In order to process your application, we will need to verify the information you provided is accurate. You will need to submit the following documents through text, email, other systems implemented by the CBO, or in-person:"
+          photo_id_accordion:
+            title: "Photo ID"
+            body: "TODO"
+          proof_of_address_accordion:
+            title: "Proof of address"
+            body: "TODO"
+          covid19_impact_accordion:
+            title: "COVID-19 impact"
+            body: "TODO"
+        application_number:
+          title: "Application Number"
+          message: "The client should write this number down. They will need it to pick up the benefit in person."
+          confirm: "Confirm that the applicant received the email or text message with the Application Number.  If not, review the contact information and make any necessary edits."
+        next_steps:
+          title: "Next Steps"
+          message_html: |
+            <ol>
```

Comment:
I thought the way the lists are numbered might vary in translation so I did each item individually and included the number in the story I'm working on now.